scapy/contrib/pnio.py

```diff
@@ -22,8 +22,19 @@
 """
 
 # Scapy imports
-from scapy.all import Packet, bind_layers, Ether, UDP
-from scapy.fields import XShortEnumField
+import copy
+from scapy.packet import Packet, bind_layers
+from scapy.packet import Packet, bind_layers
```

Comment:
Good catch. Thank you. I must have hit a shortcut to duplicate a line, in my IDE, by accident.